README.md

```diff
@@ -56,6 +56,11 @@ NODES_FILE="/root/dev-scripts/ironic_hosts.json"
 INT_IF="em2"
 # Set to the interface used by the provisioning bridge
 PRO_IF="em1"
+# This is the interface for the provisioning network on the deployed hosts.
+# This value will be placed in the ConfigMap named ironic-bmo-configmap
+# and used by the baremetal-operator pod in the cluster.  “ens3” is the
+# default.
+PROV_IF=”ens3”
```

Comment:
Is there a reason we don't update these in config_example.sh anymore?

Reply:
I would prefer we documented options there instead of the README.  In this case I just searched for PRO_IF and documented PROV_IF next to it ...

Reply:
Yea was just a general comment, it looks like we haven't updated the example in some time.